Library/Foundation/Bundle+FileResource.swift

```diff
@@ -8,16 +8,16 @@
 
 import Foundation
 
-public extension NSBundle {
+public extension Bundle {
   /**
    Returns the file URL for the given resource (R.file.*).
 
    - parameter resource: The resource to get the file URL for (R.file.*).
 
    - returns: The file URL for the resource file (R.file.*) or nil if the file could not be located.
    */
-  public func URLForResource(resource: FileResourceType) -> NSURL? {
-    return URLForResource(resource.name, withExtension: resource.pathExtension)
+  public func urlForResource(_ resource: FileResourceType) -> URL? {
```

Comment:
Should this be `url(forResource: FileResourceType)` instead?
Or is that too different from the `urlForResource(_, withExtension:)` it replaces?